microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -119,7 +119,7 @@ def verify_grub(self, node: Node) -> None:
             # in core os we don't have access to boot partition
             grub_output = node.tools[Dmesg].run().stdout
         else:
-            raise LisaException(f"Test cannot run on distro {node.os}")
+            raise SkippedException(f"Test cannot run on distro {node.os}")
```

Comment:
In our test guideline, for unknown things, they should make the test case fail. If it's confirmed not to test in Mariner, please add a `elif` to skip Mariner only.